sass/index.scss

```diff
@@ -611,24 +662,23 @@ footer {
         }
     }
 
-    footer {
-        height: 200px;
-        background-size: auto 200px;
-
-        #footer-content {
-            height: 157px;
-
-            h2 {
-                font-size: 1.2em;
-            }
+    // Contact
+    #contact #contact-socials {
+        max-width: 100%;
+        width: 100%;
+        justify-content: flex-start;
+        padding: 0;
+        margin: 2rem 0;
+        background-image: none;
```

Comment:
Clever to hide the image on mobile! easiest debugging is just removing it haha

Reply:
Not sure if this is something I am supposed to change or not

Reply:
I was commenting on how its a good solution, nothing to change here